client/components/OrderHistory.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { fetchOrders } from "../store/orders";
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+class OrderHistory extends React.Component {
+    componentDidMount() {
+        this.props.loadUserOrdersThunk();
+    }
+
+    render() {
+        return (
+            <div>
+                <div>
+                    <h1>Order History</h1>
+                    <div>
+                        {
+                            this.props.orders.map(order => {
+                                let date = order.updatedAt.slice(0, 10)
+                                return (
+                                    <div key={order.id}>
+                                        <p>Order ID: {order.id}</p>
+                                        <p>Order Date: {date[0]}</p>
+                                         {/* maybe add more */}
+                                    </div>
+                                )
+                            })
+                        }
+                    </div>
+                </div>
+            </div>
+        )
+    }
+};
+
+const mapState = (state) => {
+    return {
+        orders: state.orders
+    }
+};
+
+const mapDispatch = (dispatch) => {
+    return {
+        thunk: () => dispatch(fetchOrders())
```

Comment:
This property should probably be named something like fetchOrders or getOrders or loadOrders. In your component you want to "get the orders when the component loads," and you want your code to describe that happening. Something generic like "thunk" doesn't mean anything to a person coming in to read your code